index.html

```diff
@@ -0,0 +1,167 @@
+                    </li>
+                    <li>
+                        <a href="#venue" class="a-links">Schedule</a>
+                    </li>
+                    <li>
+                        <a href="#HOF" class="a-links">Speakers</a>
+                    </li>
+                    <li>
+                        <a href="#prospects" class="a-links">Partner</a>
+                    </li>
+                    <li>
+                        <a href="#prospects" class="a-links">Join Us!</a>
+                    </li>
+                </ul>
+
+            </nav>
+        </header>
+
+        <div class="container">
+            <P class="p-1">Sports for All: Breaking Barriers in India</P>
+            <h1 class="head-1">Sport Summit <br> India 2023</h1>
+            <p class="box-1">
+                The Sport Summit India 2023 is an exciting opportunity for sports enthusiasts, professionals, and
+                organizations to come together and discuss how we can break barriers and create more inclusive and
+                accessible sports environments in India. </p>
+            <h3 class="date">29.08.2023(TUE)</h3>
+            <p class="venue0"><i class="fa fa-map-marker" aria-hidden="true"></i> Indira Gandhi Stadium Complex, New
+                Delhi</p>
+        </div>
+    </section>
+    <section class="program">
+        <div class="head-2"><h2>Main Program</h2></div>
+        <hr class="short-hr">
+        <div class="program-list">
+            <ul class="main-pro">
+                <li>
+                    <img src="assets/opening-ceremony.png" alt="img">
+                    <h3>Opening Cermony</h3>
+                    <p>The opening ceremony celebrates the power of sports to bring people together, and it sets the
+                        stage for an exciting and unforgettable event.</p>
+                </li>
+                <li>
+                    <img src="assets/speech.png" alt="img">
+                    <h3>Speakers</h3>
+                    <p>The speakers at the event are renowned sports personalities who have made significant contributions to their respective sports.</p>
+                </li>
+                <li>
+                    <img src="assets/chat.png" alt="img">
+                    <h3>Forum</h3>
+                    <p>Have the time to share your thoughts and opinion with experts.</p>
+                </li>
+                <li>
+                    <img src="assets/ping-pong.png" alt="img">
+                    <h3>Skill-Show</h3>
+                    <p>The Skill-Show event is a testament to the incredible talent and athleticism of the world's top sportspeople.</p>
+                </li>
+                <li>
+                    <img src="assets/netoworking.png" alt="img">
+                    <h3>SS Ignite</h3>
+                    <p>Get opportunities to network with SS affiliates around the world, even after the summit. </p>
+                </li>
+            </ul>
+        </div>
```

Comment:
- [ ] Please ensure that individual program has the same dimension according to the given template, this will make your project to be more appealing. 👍 

<img width="426" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235325167-0bb7fd0e-d7ef-4ac0-a783-34c9d899a23d.png">